CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [ 1.22.1 ] - 2021-04-20
+### Changed
+- Updated travis.yml to replace Eimi's npm credentials with our Jenkins credentials
```

Comment:
Would be good to get a ref to the issue number on this line eg "I77: Updated Travis...."